documentation/release-notes/topics/version_4.md

```diff
@@ -1,5 +1,25 @@
 # Version 4
 
+## Hyperview 4.7 (TBD)
+
+This section covers significant changes and bug fixes in Hyperview 4.7.x since version 4.6.x
+
+:::{important}
+1. This release has changes and improvements to the Data Collector software. We strongly recommend updating your installed Data Collectors to the latest version to maintain an optimal monitoring and discovery experience.
+2. Support for the *Windows* version of the Data Collector will end on *January 31, 2025*. Customers are encouraged to switch to the Linux version before then. Detailed {ref}`installation instructions <Setting-up-data-collectors-doc>` are in the product documentation.
+3. Microsoft has [announced](https://devblogs.microsoft.com/microsoft365dev/retirement-of-office-365-connectors-within-microsoft-teams/) the sudden and quick retirement of the Webhook interface within Microsoft Teams. This impacts the Notifications Channels functionality. We are evaluating options and will be addressing this in an upcoming release.
+:::
+
+### Enhanced Feature: Device Discovery
+
+Discovery for Cisco switches has been enhanced to auto-detect and track power information. If detected, new sensors will be created to track the power data.
+
+Discovery for Rittal CMC III monitoring system has been improved to detect more environmental sensors.
+
+:::{important}
+These enhancements requires the latest Linux version of the Data Collector.
```

Comment:
Should be "require" instead of "requires".

Reply:
fixed